app/decorators/audit_activity/correspondence/phone_call_updated_decorator.rb

```diff
@@ -0,0 +1,30 @@
+module AuditActivity
+  module Correspondence
+    class PhoneCallUpdatedDecorator < AuditActivity::Correspondence::BaseDecorator
+      def new_correspondent_name
+        metadata.dig("updates", "correspondent_name", 1)
+      end
+
+      def new_phone_number
+        metadata.dig("updates", "phone_number", 1)
+      end
+
+      def new_correspondence_date
+        Date.parse(metadata.dig("updates", "correspondence_date", 1)).to_s(:govuk)
+      rescue StandardError # rubocop:disable Lint/SuppressedException
+      end
```

Comment:
Can we avoid the rescue by checking if the value is nil first?

```suggestion
      def new_correspondence_date        
        value = metadata.dig("updates", "correspondence_date", 1)
        return if value.nil?
        Date.parse(value).to_s(:govuk)
      end
```

Reply:
The value can still raise an exception and I'm not certain this is more readable so I'm going to keep it this way.